### PR TITLE
MBMS 61073: Self-Service: Pre-Need: Update Date Formatting

### PR DESCRIPTION
**Pull request by adam-antonioli**
# Jira Link: [MBMS-61073](https://jira.devops.va.gov/browse/MBMS-61073)

## Summary

As a Self-Service Developer I need to update the date formatting at the bottom of the Pre-Need form to match the formatting for how the date is populated in the date field, MM/DD/YYYY.

## Testing done

- Local testing

## Screenshots
Screenshot 2024-03-19 at 15 07 49

## What areas of the site does it impact?

Pre-need form 40-10007

## Acceptance criteria

Update date field on the bottom of the Pre-Need form to use a MM/DD/YYYY format

- [x]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x]  No error nor warning in the console.
- [x]  Events are being sent to the appropriate logging solution
- [x]  Documentation has been updated (link to documentation)
- [x]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x]  Feature/bug has a monitor built into Datadog or Grafana (if applicable)
- [x]  If app impacted requires authentication, did you login to a local build and verify all authenticated routes work as expected
- [x]  I added a screenshot of the developed feature

### CODE REVIEW

**Review by ksantiagoBAH — APPROVED** [review]

**Review by ConnorJeff — DISMISSED** [review]

**Review by sonntag-adam — APPROVED** [review]

File: lib/central_mail/datestamp_pdf.rb [hunk 1]
```diff
@@ -13,18 +13,23 @@ def initialize(file_path, append_to_stamp: nil)
...
 
     # rubocop:disable Metrics/ParameterLists
     # rubocop:disable Metrics/MethodLength
-    def generate_stamp(stamp_path, text, x, y, text_only, size = 10, timestamp = nil, page_number = nil, template = nil)
+    def generate_stamp(stamp_path, text, x, y, text_only, size = 10, timestamp = nil, page_number = nil,
+                       template = nil, file_path = nil)
       timestamp ||= Time.zone.now
       unless text_only
-        text += " #{I18n.l(timestamp, format: :pdf_stamp)}"
+        text += if file_path == 'tmp/vba_40_10007-stamped.pdf'
```
Comment (LindseySaari) [inline lib/central_mail/datestamp_pdf.rb:28]: Are there any specs that you can update to support this change?

Comment (adam-antonioli) [reply lib/central_mail/datestamp_pdf.rb:28]: Ok I added two test cases to `spec/lib/central_mail/datestamp_pdf_spec.rb` for this

**Review by ksantiagoBAH — APPROVED** [review]

**Review by ConnorJeff — APPROVED** [review]

**Review by LindseySaari — APPROVED** [review]

**Review by LindseySaari — APPROVED** [review]

**Review by mathisto — APPROVED** [review]
Looks good. I double-checked format on https://strftime.net
`"03/22/2024"`

**Review by caseywilliams — APPROVED** [review]
Looks fine to me from the Decision Reviews side!

**Pull request merged**